lib/order.rb

```diff
@@ -0,0 +1,66 @@
+require 'awesome_print'
+require 'csv'
+require_relative 'customer'
+
+class Order
+  attr_reader :id, :customer, :products
+  attr_accessor :fulfillment_status
+
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id #integer
+    @products = products #hash
+    @customer = customer #instance of customer
+    status_options = [:pending, :paid, :processing, :shipped, :complete]
```

Comment:
Consider making `status_options` a constant defined at the start of your program.